language_names/manual.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3
+import textwrap
+import csv
+
+
+def insert_values(c, filename):
+    c.execute(textwrap.dedent("""
+        CREATE TABLE IF NOT EXISTS {file} (
+            id INTEGER PRIMARY KEY,
+            lg TEXT,
+            inLg TEXT,
+            name TEXT,
+            UNIQUE(lg, inLg) ON CONFLICT REPLACE);
+        """.format(file=filename)))
```

Comment:
Is there a near to use keyword arguments here? Just a `{}` in the string should suffice.

Reply:
Okay it can be done using string interpolation